.vib/wildfly/goss/vars.yaml

```diff
@@ -0,0 +1,20 @@
+binaries:
+  - java
```

Comment:
I would also check the most important Wildfly's binaries; at least `standalone.sh`.